test/console-smoke/console.ts

```diff
@@ -0,0 +1,99 @@
+import { By, WebDriver } from 'selenium-webdriver'
+import {
+  clickElement,
+  clickWebElement,
+  getByDataTestID,
+  getElementText,
+  getElements,
+  getWebElementText,
+  isElementDisplayed,
+  sendKeysToElement,
+  waitForElementToBeReady,
+  waitForElementToDisappear
+} from '../e2e/helpers/selenium-util'
+import { elementIsDisabled } from 'selenium-webdriver/lib/until'
+
+export class Console {
+  private readonly connect: By = getByDataTestID('connect-vega-wallet')
+  private readonly connectToBrowserWallet: By = getByDataTestID('connector-injected')
+  private readonly orderTab: By = getByDataTestID('Order')
+  private readonly networkDropdown: By = getByDataTestID('navbar-network-switcher-trigger')
+  private readonly orderSize: By = getByDataTestID('order-size')
+  private readonly orderPrice: By = getByDataTestID('order-price')
+  private readonly placeOrder: By = getByDataTestID('place-order')
+  private readonly welcomeDialog: By = getByDataTestID('dialog-close')
+  private readonly connectDialog: By = getByDataTestID('wallet-dialog-title')
+  private readonly successToast: By = By.css("[data-testid='toast'][class*='bg-vega-green']")
+  private readonly closeAnyToast: By = By.css("[data-testid='toast-close']")
+  private readonly marketsParentElement: By = By.css("[data-testid*='pathname-/markets']")
+  private readonly marketSelector: By = getByDataTestID('popover-trigger')
+  private readonly marketSelectorPrice: By = By.css("[data-testid='market-selector-price']")
+
+  constructor(private readonly driver: WebDriver) {}
+
+  async checkTransactionSuccess() {
+    expect(await isElementDisplayed(this.driver, this.successToast), 'expected to see a success toast', {
+      showPrefix: false
+    }).toBe(true)
+  }
+
+  async connectToWallet() {
+    console.log('clicking connect')
```

Comment:
nitpick:

stray logs here and 2 lines down?

Reply:
..yes :D